tests/models/test_lora_layers.py

```diff
@@ -467,3 +454,98 @@ def test_lora_save_load_with_xformers(self):
 
         # Outputs shouldn't match.
         self.assertFalse(torch.allclose(torch.from_numpy(orig_image_slice), torch.from_numpy(lora_image_slice)))
+
+
+@slow
+@require_torch_gpu
+class LoraIntegrationTests(unittest.TestCase):
+    def test_dreambooth_old_format(self):
+        generator = torch.Generator("cpu").manual_seed(0)
+
+        lora_model_id = "hf-internal-testing/lora_dreambooth_dog_example"
+        card = RepoCard.load(lora_model_id)
+        base_model_id = card.data.to_dict()["base_model"]
+
+        pipe = StableDiffusionPipeline.from_pretrained(base_model_id, safety_checker=None)
+        pipe = pipe.to(torch_device)
+        pipe.load_lora_weights(lora_model_id)
+
+        images = pipe(
+            "A photo of a sks dog floating in the river", output_type="np", generator=generator, num_inference_steps=2
+        ).images
+
+        images = images[0, -3:, -3:, -1].flatten()
+
+        expected = np.array(
+            [
+                0.7207,
+                0.6787,
+                0.6010,
+                0.7478,
+                0.6838,
+                0.6064,
+                0.6984,
+                0.6443,
+                0.5785,
+            ]
```

Comment:
(nit): Could we reduce the LoC like we're doing here?
https://github.com/huggingface/diffusers/pull/3778/files#diff-addc76c32ca1ded9bcc8be9ad7e8c44178737df9821dd529e973828486c3ae5bR510

Reply:
done!